docs/docusaurus/docs/reference/learn/data_quality_use_cases/integrity.md

```diff
@@ -0,0 +1,278 @@
+
+### Expect Column Pair Values To Be Equal
+
+Validates that values in one column match corresponding values in another column.
+
+**Use Case**: Verify that the reference number and confirmation code match for each transaction.
+
+```python
+gxe.ExpectColumnPairValuesToBeEqual(
+    column_A="reference_number",
+    column_B="confirmation_code"
+)
+```
+
+<small>View `ExpectColumnPairValuesToBeEqual` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_pair_values_to_be_equal).</small>
+
+
+### Expect Multicolumn Sum To Equal
+
+Ensures that the sum of multiple columns equals an expected value, useful for validating financial calculations and balance checks.
+
+**Use Case**: Validate sum-to-zero checks in the `transfer_balances` table.
+
+```python
+gxe.ExpectMulticolumnSumToEqual(
+    column_list=["adjustment", "sender_debit", "recipient_credit"],
+    sum_total=0.0
+)
+```
+
+<small>View `ExpectMulticolumnSumToEqual` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_multicolumn_sum_to_equal).</small>
+
+
+### Expect Column Pair Values A To Be Greater Than B
+
+Verifies that values in one column are consistently greater than related values in another column.
+
+**Use Case**: Ensure transfer amounts are always greater than or equal to adjustments.
+
+```python
+gxe.ExpectColumnPairValuesAToBeGreaterThanB(
+    column_A="transfer_amount",
+    column_B="adjustment",
+    or_equal=True
+)
+```
+
+<small>View `ExpectColumnPairValuesAToBeGreaterThanB` in the [Expectation Gallery](https://greatexpectations.io/expectations/expect_column_pair_values_a_to_be_greater_than_b).</small>
+
+<br/>
+<br/>
+
+:::tip[GX tips for integrity Expectations]
+- Use custom SQL Expectations when built-in Expectations don't cover complex business rules
+- Combine multiple Expectations to create comprehensive integrity checks
+- Consider performance implications when validating across large datasets
+:::
+
+## Example: Validate cross-table balance integrity
+
+**Context**: In this example, we'll use the sample `transactions` and `transfer_balances` tables to demonstrate how to validate the integrity of transfer amounts, adjustments, and balances between related tables. Ensuring consistency across these tables is crucial for accurate financial reporting and auditing.
+
+**Goal**: Using GX Cloud or GX Core, implement integrity validations that:
+1. Ensure `transaction_id` is consistent between the `transactions` and `transfer_balances` tables
+2. Verify that the `transfer_amount` in `transfer_balances` matches the absolute value of `sender_debit` for each transaction
+3. Validate that `recipient_credit` equals `transfer_amount` plus `adjustment` for each record in `transfer_balances`
+
+<Tabs
+   defaultValue="gx_cloud"
+   values={[
+      {value: 'gx_cloud', label: 'GX Cloud'},
+      {value: 'gx_core', label: 'GX Core'}
```

Comment:
I haven't had a chance to dig into the GX Core side of this, but I'll take the action on writing the code example.